tests/main/uc20-create-partitions-reinstall/task.yaml

```diff
@@ -1,8 +1,14 @@
-summary: Integration tests for the snap-bootstrap binary autodetect
+summary: Integration tests for the bootstrap.Run autodetect
 
 # use the same system and tooling as uc20
 systems: [ubuntu-20.04-64]
 
+environment:
+    # an empty $topsrcdir/tests/go.mod seems to break importing or building go
+    # packages referenced by their import paths while under the tests directory,
+    # need to disable go modules support for this test
+    GO111MODULE: off
```

Comment:
this is both setting this and removing go.mod, I suppose go.mod should be reinstated?

Reply:
Sorry for that, not sure how this happend